src/python/pants/backend/python/dependency_inference/module_mapper_test.py

```diff
@@ -418,11 +441,85 @@ def req(
                 ),
                 "default": FrozenDict(
```

Comment:
**comment:** the ordering of these are super annoying!
in general, this was a pain to test as the output doesn't easily indicate what is wrong. so, had to copy/paste the output, then format manually, then compare.
most of the time it was ordering problems. it's in alphabetical order. was tempted to wrap this in `ordered` or something but didn't.

just thought i'd share some feedback and solicit any advice/ideas on comparing nested objects so the output is better.